Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,87 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return "";
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfLines = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+            var lengthLongestWord = 0;
+
+            if (arrayOfLines.Length == 1)
+            {
+                lengthLongestWord = arrayOfLines[0].Length;
+            }
+            else
+            {
+                foreach (var word in arrayOfLines)
+                {
+                    if (word.Length > lengthLongestWord)
+                    {
+                        lengthLongestWord = word.Length;
+                    }
+                }
+            }
+
+            // print table 
+
+            var table = new StringBuilder() ; 
+            // print header
+            table = DisplayTopOrBottomLine(padding, lengthLongestWord, table);
+
+            // print empty lines
+            table = DisplayEmptyLines(padding, lengthLongestWord, table);
+
+            // print text 
+
+            foreach (var word in arrayOfLines)
+            {
+                table.Append( "|");
+                table.Append (String.Empty.PadRight(padding, ' '));
+                table.Append(word);
+                table.Append(String.Empty.PadRight(lengthLongestWord - word.Length + padding, ' '));
+                table.Append("|");
+                table.Append(Environment.NewLine);
+
+            }
```

Comment:
This could be a function, too. However I think it's fine to keep it like that.

Reply:
oke, I could also change that but then I have to think of a good name of that function. Maybe DisplayLinesWithWord ?